src/models/tools/tool-content-info.ts

```diff
@@ -22,16 +23,25 @@ export interface IDefaultContentOptions {
   unit?: UnitModelType;
 }
 
+type ToolComponentType = React.ComponentType<IToolTileProps> & {
+  tileHandlesSelection?: boolean;
+  // This doesn't seem to be implemented anywhere, but perhaps it is added by a
+  // 3rd party library
+  getDragImageNode?: (dragElt: HTMLElement) => HTMLElement;
```

Comment:
This could be commented out where it is used.  And then could be removed from here.